src/internal/analytics/selectors.ts

```diff
@@ -1,17 +1,12 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import breadcrumbGroupItemStyles from '../../breadcrumb-group/item/styles.css.js';
-import breadcrumbGroupStyles from '../../breadcrumb-group/styles.css.js';
-
-export const getBreadcrumbLinkSelector = (index: number) =>
-  `.${breadcrumbGroupStyles['breadcrumb-group']} .${breadcrumbGroupStyles.item}:nth-child(${index}) .${breadcrumbGroupItemStyles.anchor}`;
```

Comment:
These class names are random and may resolve incorrectly, if breadcrumb on a page runs from a different version than this code.

Replaced with a data-attribute marker, which is more reliable